stable/airflow/values.yaml

```diff
@@ -1167,6 +1167,10 @@ rbac:
   ##   which is required for the KubernetesPodOperator() to function
   ##
   create: true
+  ## If the created role has access to GET and LIST `events` resources: you will need
+  ## this if for example you set `log_events_on_failure=True` when using airflow's
+  ## KubernetesPodExecutor
+  events: false
```

Comment:
Can you please format the doc string the same as other values?

```yaml
...
rbac:
  ## if Kubernetes RBAC resources are created
  ##
  ## NOTE:
  ## - these allow the service account to create/delete Pods in the airflow namespace,
  ##   which is required for the KubernetesPodOperator() to function
  ##

  ## if the created RBAC Role has GET/LIST on event Resources
  ## 
  ## NOTE:
  ## - this is needed for KubernetesPodOperator() to use `log_events_on_failure=True`
  ##
  events: false
```

Reply:
👍